modules/nf-core/sourmash/index/meta.yml

```diff
@@ -0,0 +1,49 @@
+name: "sourmash_index"
+description: Create a database of sourmash signatures (a group of FracMinHash sketches) to be used as references.
+keywords:
+  - signatures
+  - sourmash
+  - genomics
+  - metagenomics
+  - mapping
+  - kmer
+tools:
+  - "sourmash":
+      description: "Compute and compare FracMinHash signatures for DNA data sets."
+      homepage: "https://sourmash.readthedocs.io/"
+      documentation: "https://sourmash.readthedocs.io/"
+      tool_dev_url: "https://github.com/sourmash-bio/sourmash"
+      doi: "10.21105/joss.00027"
+      licence: "['BSD-3-clause']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - signatures:
+      type: file
+      description: Files containing signature (hash sketches) of reference genomes
+      pattern: "*.{sig}"
+  - ksize:
```

Comment:
Remove this, not in `main.nf`

Reply:
You mean `ksize`, I suppose? Can I keep it but add that it should be supplied with `ext.args`? Or should I add it back to `main.nf` but not as a pre-defined value? (Sorry for all the questions, haven't written many modules.)

Reply:
Yes, the ksize key should be removed.
You can add a comment to `main.nf` under `ext.args` that the module needs the `ksize` key to run.